scripts/functions.sh

```diff
@@ -3,6 +3,10 @@
 
 export request="wget --progress=bar:force:noscroll -c --tries=2 "
 
+function log() {
+    echo "$0:${BASH_LINENO[*]}": $@
```

Comment:
The column number will always be `0`, so we only need the line number from the `BASH_LINENO` array, I think?

Reply:
Currently:
```
shoreline-geoserver-1   | /scripts/entrypoint.sh:16 0: CLUSTER_CONFIG_DIR=/data/cluster/instance_
shoreline-geoserver-1   | /scripts/entrypoint.sh:17 0: MONITOR_AUDIT_PATH=/data/monitoring/monitor_
```